hosting/src/events/Events/Events.ts

```diff
@@ -1,14 +1,24 @@
-import { Vue } from 'vue-class-component'
+import dayjs from 'dayjs'
+import { Options, Vue } from 'vue-class-component'
 import { useMeta } from 'vue-meta'
 
 import { Route } from '~/router/route-decorator'
 import { EventUi } from '../models'
 
+import EventRow from './EventRow.vue'
+
 @Route({ path: '/events' })
+@Options({ components: { EventRow } })
 export default class Events extends Vue {
   meta = useMeta({ title: 'Events' })
 
-  get events(): EventUi[] {
-    return this.$store.getters.events
+  get currentEvents(): EventUi[] {
+    const now = dayjs()
+    return this.$store.getters.events.filter((event: EventUi) => event.dateStart.isAfter(now)).reverse()
+  }
+
+  get pastEvents(): EventUi[] {
+    const now = dayjs()
+    return this.$store.getters.events.filter((event: EventUi) => event.dateStart.isBefore(now))
   }
```

Comment:
We might consider making these Vuex getters.
`Home` should be using `futureEvents` too.

Reply:
Yep we should probably load them from Firebase independently too.

Reply:
I did look at doing that but then I would have had to also change the way we get the event detail page using the slug.